src/main/java/dk/kb/netarchivesuite/solrwayback/parsers/WarcParser.java

```diff
@@ -267,7 +265,7 @@ else if (headerLine.toLowerCase().startsWith("location:")) {
     }
     else if (headerLine.toLowerCase().startsWith("transfer-encoding:")) {                                      
       String transferEncoding=headerLine.substring(18).trim();
```

Comment:
Why use `substring` here when `split` is used on line 255? Consistent solutions to the same problem makes the code easier to real. Also, the use of constants such as `18` makes it harder to determine if the code does what it is supposed to.

Reply:
Because transfer encoding can contain the ':' character as well.